src/pagination/pagination-model.class.ts

```diff
@@ -7,7 +7,7 @@ export class PaginationModel {
 	/**
 	 * Length of page.
 	 */
-	pageLength = 10;
+	pageLength ? = 10;
```

Comment:
This is marking the property as optional right? So I think we may want to format it like this:

```suggestion
	pageLength? = 10;
```

Reply:
I found that the linter didn't like that

Reply:
Huh, interesting ... We'll take a look at that after this is done 🤔